charts/images.yaml

```diff
@@ -175,6 +175,10 @@ images:
   sourceRepository: github.com/elastic/elasticsearch-docker
   repository: docker.elastic.co/elasticsearch/elasticsearch-oss
   tag: "6.5.4"
+- name: elasticsearch-searchguard-oss
+  sourceRepository: github.com/floragunncom/search-guard
+  repository: floragunncom/sg-elasticsearch
```

Comment:
are we sure this repository is trustworthy. Does not look like an official repository.

Reply:
I think so. We had a discussions with the team about that and we got a decision to use the existing repository and not manage our own. The repo is given in the searchguard examples, so i hope it is trusted.